validator/validator.protoascii

```diff
@@ -2373,6 +2373,30 @@ tags: {  # amp-pinterest
     }
   }
 }
+tags: {  # amp-reach-player
```

Comment:
@Gregable you ok with this getting merged externally? let me know, since might LGTM this pull request soon. Thanks!